Google-Summer-of-Code-2024.md

```diff
@@ -205,13 +204,15 @@ When submitting a proposal, please use the provided GSoC proposal template. We w
 
 **Note:** although some of the previous years' proposals are a bit on the long side, there's **no** formal length requirement for your proposal. The quality of what you write is much more important than the amount of text you write, and we encourage you to write shorter proposals that still convey the main aim of the project.
 
-
-
 (**NOTE**: The link to the 2024 template will be posted soon. **It will differ from the 2023 template.** We will post instructions together with the template.)
 
 **Some important notes:**
 
-1. When the necessary criteria for requesting a review are met, share your proposal to gsoc-2024-mentors@oppia.org and provide them with commenting permissions. Use only this channel for proposal reviews; **do not** send proposals directly to individual GSoC mentors. All proposal-related communication should happen through the gsoc-2024-mentors@oppia.org mailing list or directly through comments in the proposal doc.
+1. When the necessary criteria for requesting a review are met, add gsoc-2024-mentors@oppia.org as an editor for your proposal doc. (This makes some workflows, like inviting PMs or fixing typos, etc., easier, but if you're concerned about changes to your doc, then you can [turn on notifications for edits](https://support.google.com/docs/answer/91588?hl=en&co=GENIE.Platform%3DDesktop).) After fixing the sharing settings, make a new post in the correct "proposal reviews" category in [GitHub Discussions](https://github.com/oppia/oppia/discussions) that is clearly titled with the name of the project that you are requesting a review for, and provide a link to the doc in your post.
+
+Please use only the above channel for proposal reviews: all proposal-related communication should happen through GitHub Discussions or directly through comments in the proposal doc. **Do not** send proposals directly to individual GSoC mentors.
+
+You can also request **at most one** "tech lead review" for **at most one** of your proposals during the pre-selection phase. To keep things fair, the tech lead will do only a single pass on your proposal and leave comments, but is not required to follow up on replies to those comments. Since you can only request a tech lead review once (per applicant), we recommend doing so after you have gotten feedback from mentors and completed a full draft of your proposal, but at least a week before the due date. Tech leads will process requests in the order they are received. To request a tech lead review, fill in [this Google Form](https://forms.gle/v6QCE4KrEt1vRRKv8).
```

Comment:
Why `at most one of your proposals`?? Shouldn't we do this for each proposal they submit?(Incase if they request)

Reply:
Tech leads won't have enough bandwidth, and we will only accept at most one proposal from each applicant anyway. Since this is the first year we are doing this, I think it would be better to keep it light on the tech leads otherwise we make promises we might not be able to fulfill.